battlepass/src/types.rs

```diff
@@ -0,0 +1,34 @@
+use frame_support::pallet_prelude::*;
+use codec::MaxEncodedLen;
+
+#[derive(Encode, Decode, Clone, PartialEq, Eq, PartialOrd, Ord, TypeInfo, MaxEncodedLen)]
+pub enum BattlepassState {
+	Draft = 0,
+	Active = 1,
+	Closed = 2,
```

Comment:
`Closed` => `Ended`
`Locked`

Reply:
maybe `Closed` -> `Finished`? And the extrinsic accordingly `finish_battlepass`?
Why do we need `Locked`? @2075